src/components/notifications/SendGrid.vue

```diff
@@ -0,0 +1,35 @@
+<template>    
+    <!-- SendGrid fields -->
+    <div class="mb-3">
+      <label for="sendgrid-api-key" class="form-label">{{ $t("SendGrid API Key") }}</label>
+      <input id="sendgrid-api-key" v-model="$parent.notification.sendgridApiKey" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+      <label for="sendgrid-from-email" class="form-label">{{ $t("From Email") }}</label>
+      <input id="sendgrid-from-email" v-model="$parent.notification.sendgridFromEmail" type="email" class="form-control" required>
+    </div>
+    <div class="mb-3">
+      <label for="sendgrid-to-email" class="form-label">{{ $t("To Email") }}</label>
+      <input id="sendgrid-to-email" v-model="$parent.notification.sendgridToEmail" type="email" class="form-control" required>
+    </div>
+    <div class="mb-3">
+      <label for="sendgrid-cc-email" class="form-label">{{ $t("CC Email") }}</label>
+      <input id="sendgrid-cc-email" v-model="$parent.notification.sendgridCcEmail" type="email" class="form-control">
+      <small class="form-text text-muted">{{ $t("Separate multiple email addresses with commas") }}</small>
+    </div>
+    <div class="mb-3">
+      <label for="sendgrid-bcc-email" class="form-label">{{ $t("BCC Email") }}</label>
+      <input id="sendgrid-bcc-email" v-model="$parent.notification.sendgridBccEmail" type="email" class="form-control">
+      <small class="form-text text-muted">{{ $t("Separate multiple email addresses with commas") }}</small>
+    </div>
+    <div class="mb-3">
+      <label for="sendgrid-subject" class="form-label">{{ $t("Email Subject") }}</label>
+      <input id="sendgrid-subject" v-model="$parent.notification.sendgridSubject" type="text" class="form-control" required>
+      <small class="form-text text-muted">{{ $t("Default subject will be used if left empty") }}</small>
```

Comment:
The input is marked as `required`, which conflicts with `if left empty`..

Reply:
done [8b015a6](https://github.com/louislam/uptime-kuma/pull/5205/commits/8b015a6c4a021e8062ccb5cd0ffc379ac9c759d0)